package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
why is there a package-lock?
probably sneaked in there.
do you use npm or yarn? please use yarn

Reply:
I don't know why there's a package-lock, but I will only use yarn from now on